src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -1,4 +1,4 @@
-# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model project <dynamic.grid.calculation@alliander.com>
+# SPDX-FileCopyrightText: 2022 Contributors to the Power Grid Model IO project <dynamic.grid.calculation@alliander.com>
```

Comment:
You don't need to add IO. The project is still Power Grid Model. `power-grid-model-io` is just one of the repositories and libraries.